### PR TITLE
Add Bitcoin to AnySigner

### PR DESCRIPTION
**Pull request by EnoRage**
## Description
Bitcoin to AnySigner was added

## Checklist

- [x] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

**Comment by hewigovens** [comment]
@EnoRage hey Nick, could you please create a clean pr? a few unrelated changes in the diff

**Comment by EnoRage** [comment]
@hewigovens Sorry, I removed everything except Any Signer for Bitcoin, must be ok right now

File: tests/interface/TWAnySignerTests.cpp [hunk 1]
```diff
@@ -21,7 +21,6 @@ TEST(TWAnySigner, IsEnabled) {
     ASSERT_TRUE(TWAnySignerIsSignEnabled(TWCoinTypeBinance));
 
     ASSERT_FALSE(TWAnySignerIsSignEnabled(TWCoinTypeBitcoinCash));
-    ASSERT_FALSE(TWAnySignerIsSignEnabled(TWCoinTypeBitcoin));
```
Comment (hewigovens) [inline tests/interface/TWAnySignerTests.cpp]: just change to `ASSERT_TRUE`

Comment (EnoRage) [reply tests/interface/TWAnySignerTests.cpp]: Done

**Pull request merged**